### DISCUSSION TITLE
ajouter des potions d'énergie

### DISCUSSION THREAD

**Discussion by symsym-1629** [Crownicles/Crownicles#1341] [Idées acceptées / déjà implémentées]
On pourrait ajouter des potions qui te restorent une partie de ton énergie après un fight (utilisables avec /drink)

**Comment by Pietagorh** [comment]
Ou alors on donne la possibilité aux gens d'utiliser la bouffe de guilde pour s'en redonner :doge:
Comme ça les gens qui ont déjà des pets dressés arrêteront de se plaindre
Et puis c'est pas trop illogique x)

**Reply by Pietagorh** [reply]
P'tit /eat là miam
